src/katsdpcontroller/generator.py

```diff
@@ -1513,10 +1489,140 @@ def get_baseline_index(a1, a2):
         )
         xbgpu.static_gauges["xbgpu_expected_engines"] = 1.0
 
-    _add_task_sensors(g, streams, task_names)
     return xbgpu_group
 
 
+def _make_vgpu(
+    g: networkx.MultiDiGraph,
+    configuration: Configuration,
+    streams: Iterable[product_config.GpucbfTiedArrayResampledVoltageStream],
```

Comment:
We can just take a single stream here for now. We don't currently have any plans to generate multiple tarv streams from one engine.